Makefile

```
@@ -125,18 +128,23 @@ $(C_DIR_LIB):
 $(C_DIR_INCLUDE):
 	mkdir -p $(C_DIR_INCLUDE)
 
-$(C_LIBQISKIT): $(C_DIR_LIB)  $(C_LIB_CARGO_PATH)
-	cp $(C_LIB_CARGO_PATH) $(C_DIR_LIB)/$(subst _cext,,$(C_LIB_CARGO_FILENAME))
+$(C_LIBQISKIT): $(C_DIR_LIB)  $(C_LIB_RELEASE_CARGO_PATH)
+	cp $(C_LIB_RELEASE_CARGO_PATH) $(C_DIR_LIB)/$(subst _cext,,$(C_LIB_CARGO_FILENAME))
 
-$(C_QISKIT_H): $(C_DIR_INCLUDE) $(C_LIB_CARGO_PATH) 
+# This depends on either C_LIB_DEBUG_CARGO_PATH or C_LIB_RELEASE_CARGO_PATH to
+# build the original header.
+#
+# Do not depend on this in a target without one of those as dependencies first
+$(C_QISKIT_H): $(C_DIR_INCLUDE) $(C_LIB_CARGO_PATH)
```

`C_LIB_CARGO_PATH` does not exist 🙂 Or did you want to put another target here?
```suggestion
$(C_QISKIT_H): $(C_DIR_INCLUDE)
```